ngraph/core/include/ngraph/ngraph_visibility.hpp

```diff
@@ -25,11 +25,15 @@
 #pragma warning(disable : 4275)
 #endif
 
+#ifdef NGRAPH_STATIC_LIBRARY // defined if we are building or calling NGRAPH as a static library
+#define NGRAPH_API
+#else
 #ifdef ngraph_EXPORTS // defined if we are building the NGRAPH DLL (instead of using it)
 #define NGRAPH_API NGRAPH_HELPER_DLL_EXPORT
 #else
 #define NGRAPH_API NGRAPH_HELPER_DLL_IMPORT
```

Comment:
@alalek will it work if we remove `NGRAPH_HELPER_DLL_IMPORT` here? I see that OpenCV does not use import explicitly.

Reply:
Removing dllimport fails to link global variables when building the solution in shared mode.

Reply:
Ok, looks like we have to keep public definition for ngraph to denote that it's compiled statically.